### DISCUSSION TITLE
想請教字幕軌預設選擇内封字幕問題

### DISCUSSION THREAD

**Discussion by snoopyzx963** [hooke007/mpv_PlayKit#508] [Q&A 问答] [answered]
先感謝大大，受益良多，因 potplayer 改版造成特效亂碼昨日入坑

想請教内封字幕選擇預設字幕軌問題
如下 : 
例如我想預設選擇 TC，繼續播放下一集都能維持字幕軌的選擇，但當我重新啟動 mpv 之後，
設定又回歸到預設值，自動載入 SC 字幕軌
![image](https://github.com/user-attachments/assets/01b9152e-65de-4afd-ac5b-4b8e1be1ca44)

試過的方式有在 mpv.conf 更改以下指令但無效，不會預設載入 TC 字幕軌
![image](https://github.com/user-attachments/assets/e8d005c1-3785-4c70-a599-2d0a548b436e)

想請教看看是否有解決方法，謝謝!!

**Comment by hooke007** [comment]
一个都没匹配上，你的内封的语言代码是chi

**Reply by snoopyzx963** [reply]
哇~ 原來是這樣，那只好手動更改字幕軌了，謝謝!!

**Reply by hooke007** [reply, marked as answer]
封装人通常会滥用轨道标题而不是正确使用语言代码来区分（或者他根本不知道应该这么用）

ref https://github.com/mpv-player/mpv/issues/6885 感兴趣可以自己写脚本来匹配track标题解决

**Reply by snoopyzx963** [reply]
謝謝~ 日後有問題再來請教您
